### PR TITLE
Added jira issue number to print when a ticket is closed.

### LINKED ISSUE #233: Jira added debugging in logs for closing issues
### Describe the problem

> Right now when looking through the logs of Cx-Flow when a issues is closed in Jira it only shows the issues name. This makes it hard for our developers to quickly check Jiraon the issues that is closed. If the log printed out the issue number it would make it easier for devs to find the issue.

### Proposed solution

> Adding the ticket number with in the log output of closing a ticket.

### PR DESCRIPTION
**Pull request by Dinnerspy**
By submitting a PR to this repository, you agree to the terms within the [Checkmarx Code of Conduct](https://github.com/checkmarx-ts/open-source-template/blob/master/CODE-OF-CONDUCT.md). Please see the [contributing guidelines](https://github.com/checkmarx-ltd/open-source-template/blob/master/CONTRIBUTING.md) for how to create and submit a high-quality PR for this repo.

### Description

> Add the ability to easily see which Jira issues where closed in the logs of Cx-Flow by including the ticket number.

### References

Fixes #233 

### Testing

> Testing was done by running  CX-Flow and making sure the new information showed up in the logs when closing a issue.

### Checklist

- [X] I have added documentation for new/changed functionality in this PR (if applicable).  *If documentaiton is a Wiki Update, please indicate desired changes within PR MD Comment*
- [ ] All active GitHub checks for tests, formatting, and security are passing
- [X] The correct base branch is being used

### CODE REVIEW

File: src/main/java/com/checkmarx/flow/service/JiraService.java [hunk 1]
```diff
@@ -1131,14 +1131,14 @@ private void closeIssueInCaseNotWithinResults(ScanRequest request, Map<String, S
             try {
                 if (!map.containsKey(jiraIssue.getKey()) && (request.getBugTracker().getOpenStatus().contains(jiraIssue.getValue().getStatus().getName()))) {
                     /*Close the issue*/
-                    log.info("Closing issue with key {}", jiraIssue.getKey());
+                    log.info("Closing issue "+ jiraIssue.getValue().getKey()+" with key {}", jiraIssue.getKey());
```
Comment (kmcdon83) [inline src/main/java/com/checkmarx/flow/service/JiraService.java:1134]: Please use {} to introduce additional information in a log entry.  ie. log.info("{} - {}", a, b);

File: src/main/java/com/checkmarx/flow/service/JiraService.java [hunk 2]
```diff
@@ -1131,14 +1131,14 @@ private void closeIssueInCaseNotWithinResults(ScanRequest request, Map<String, S
...
                     this.transitionCloseIssue(jiraIssue.getValue().getKey(),
                             request.getBugTracker().getCloseTransition(), request.getBugTracker(), false); //No false positives
                     closedIssues.add(jiraIssue.getValue().getKey());
 
                 }
             } catch (HttpClientErrorException e) {
-                log.error("Error occurred while processing issue with key {}", jiraIssue.getKey(), e);
+                log.error("Error occurred while processing issue "+ jiraIssue.getValue().getKey()+" with key {}", jiraIssue.getKey(), e);
```
Comment (kmcdon83) [inline src/main/java/com/checkmarx/flow/service/JiraService.java:1141]: Please use {} to introduce additional information in a log entry.  ie. log.info("{} - {}", a, b);

**Comment by kmcdon83** [comment]
@Dinnerspy as per our contribution guidelines.  Please ensure an issue is created and accepted by our team before raising a PR.

**Comment by Dinnerspy** [comment]
Issue was created here: https://github.com/checkmarx-ltd/cx-flow/issues/233

**Comment by kmcdon83** [comment]
@Dinnerspy , changes look good.  Can you please rename your branch to have the prefix pr-?  This will initiate our pre-merge pipeline steps.
i.e. pr-feature/AddedLoggingOnJiraCloseIssue

**Pull request closed without merging**